### PR TITLE
Update dockerfile_functions.sh, typo in ppc64le

### PR DESCRIPTION
**Pull request by srbala**
Fixed typo in dockerfile_functions.sh

Signed-off-by: Bala Raman <srbala@gmail.com>

### CODE REVIEW

**Comment by tianon** [comment]
I don't think this one is a typo -- this `case` statement ends up embedded in the `Dockerfile` (I think) where `ppc64el` is what `dpkg --print-architecture` will give on a Debian or Ubuntu environment. :see_no_evil:

**Comment by srbala** [comment]
@tianon thanks for the note. Maybe check for both like 'ppc64el|ppc64le'? `uname -m` returns `ppc64le`

```sh
% docker run --rm -it --platform=linux/ppc64le -v $PWD:/work ubuntu
root@afc3d093095d:/# uname -m
ppc64le
root@afc3d093095d:/# dpkg --print-architecture
ppc64el
```

**Comment by srbala** [comment]
`arch` and `lscpu` returns in CentOS and AlmaLinux. Referring to old code https://github.com/AdoptOpenJDK/openjdk-docker/blob/master/dockerfile_functions.sh#L376 has both options., updated commit

File: dockerfile_functions.sh [hunk 1]
```diff
@@ -371,7 +371,7 @@ print_java_install_pre() {
 		elif [ "${sarch}" == "ppc64le" ]; then
 			JAVA_URL=$(get_v3_url feature_releases "${bld}" "${vm}" "${pkg}" ppc64le "${osfamily}");
 			cat >> "$1" <<-EOI
-       ppc64el|powerpc:common64) \\
+       ppc64el|ppc64le|powerpc:common64) \\
```
Comment (karianna) [inline dockerfile_functions.sh:374]: Comment as to why please :-)

**Review by karianna — CHANGES_REQUESTED** [review]

**Pull request closed without merging**